packages/nc-gui/components/smartsheet/Form.vue

```diff
@@ -845,8 +845,7 @@ watch(view, (nextView) => {
 }
 
 .nc-input {
-  @apply appearance-none w-full !bg-white rounded px-2 py-2 my-2 border-solid border-1 border-primary border-opacity-50;
-
+  @apply appearance-none w-full !bg-white rounded px-2 py-2 my-2 border-solid border-1 border-primary border-opacity-50 overflow-auto;
```

Comment:
unrelated changes.

Reply:
Hi, I haven't made these changes intentionally a pr generated by me previously need these changes and i updated them there only.